github/PullRequest.py

```diff
@@ -540,217 +453,177 @@ def create_review(
-        :rtype: :class:`github.PullRequestComment.PullRequestComment`
         """
         assert isinstance(id, int), id
         headers, data = self._requester.requestJsonAndCheck("GET", f"{self._parentUrl(self.url)}/comments/{id}")
         return github.PullRequestComment.PullRequestComment(self._requester, headers, data, completed=True)
 
     def get_comments(
         self,
-        sort=github.GithubObject.NotSet,
-        direction=github.GithubObject.NotSet,
-        since=github.GithubObject.NotSet,
-    ):
+        sort: Opt[str] = NotSet,
+        direction: Opt[str] = NotSet,
+        since: Opt[datetime] = NotSet,
+    ) -> PaginatedList[github.PullRequestComment.PullRequestComment]:
         """
         Warning: this only returns review comments. For normal conversation comments, use get_issue_comments.
 
         :calls: `GET /repos/{owner}/{repo}/pulls/{number}/comments <https://docs.github.com/en/rest/reference/pulls#review-comments>`_
         :param sort: string 'created' or 'updated'
         :param direction: string 'asc' or 'desc'
         :param since: datetime
-        :rtype: :class:`github.PaginatedList.PaginatedList` of :class:`github.PullRequestComment.PullRequestComment`
         """
         return self.get_review_comments(sort=sort, direction=direction, since=since)
 
     # v3: remove *, added here to force named parameters because order has changed
     def get_review_comments(
         self,
         *,
-        sort=github.GithubObject.NotSet,
-        direction=github.GithubObject.NotSet,
-        since=github.GithubObject.NotSet,
-    ):
+        sort: Opt[str] = NotSet,
+        direction: Opt[str] = NotSet,
+        since: Opt[datetime] = NotSet,
+    ) -> PaginatedList[github.PullRequestComment.PullRequestComment]:
         """
         :calls: `GET /repos/{owner}/{repo}/pulls/{number}/comments <https://docs.github.com/en/rest/reference/pulls#review-comments>`_
         :param sort: string 'created' or 'updated'
         :param direction: string 'asc' or 'desc'
         :param since: datetime
-        :rtype: :class:`github.PaginatedList.PaginatedList` of :class:`github.PullRequestComment.PullRequestComment`
-        """
-        assert sort is github.GithubObject.NotSet or isinstance(sort, str), sort
-        assert direction is github.GithubObject.NotSet or isinstance(direction, str), direction
-        assert since is github.GithubObject.NotSet or isinstance(since, datetime), since
-        url_parameters = dict()
-        if sort is not github.GithubObject.NotSet:
-            url_parameters["sort"] = sort
-        if direction is not github.GithubObject.NotSet:
-            url_parameters["direction"] = direction
-        if since is not github.GithubObject.NotSet:
+        """
+        assert sort is NotSet or isinstance(sort, str), sort
+        assert direction is NotSet or isinstance(direction, str), direction
+        assert since is NotSet or isinstance(since, datetime), since
```

Comment:
```suggestion
        assert is_optional(sort, str), sort
        assert is_optional(direction, str), direction
        assert is_optional(since, datetime), since
```